beacon-chain/sync/context.go

```diff
@@ -1,15 +1,18 @@
 package sync
 
 import (
-	"errors"
-
 	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/beacon-chain/blockchain"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
 	"github.com/prysmaticlabs/prysm/beacon-chain/p2p"
 )
 
+// Specifies the fixed size context length.
+const digestLength = 4
```

Comment:
Are `context` and `digest` interchangeable terms?

Reply:
Yeap, they are. The spec refers to a provided context which is just the fork digest.